ToDoList-frontend/src/app/bucket/bucket.component.ts

```diff
@@ -197,13 +242,13 @@ export class BucketComponent implements OnInit {
     this.currentBucketTask.bucketId = this.currentBucketId;
     this.currentBucketTask.bucketTaskStateId = this.bucketTaskStates.find(bts => bts.name === newBucketTask.bucketTaskState).id;
     this.currentBucketTask.bucketTaskPriorityId = this.bucketTaskPriorities.find(btps => btps.name === newBucketTask.bucketTaskPriority).id;
+    this.currentBucketTask.assigneeId = this.assignees.find(a => a.name === this.editNewBucketTaskFormGroup.value.assigneeControl).id;
     this.currentBucketTask.bucketTaskState = null;
     this.currentBucketTask.bucketTaskPriority = null;
 
     this.bucketTaskService.putBucketTask(environment.bucketTaskEndpoint+this.bucketTaskForEditSaveId, this.currentBucketTask).subscribe(
       (response: any) => {
         this.toastr.success("Bucket task " + this.currentBucketTask.name + " changes saved successfully.");
```

Comment:
Bardzo ladnie ze mamy toasta po pomyslnym zakonczeniu operacji. JS (a tym bardziej TS) dorobil sie jednak interpolacji stringow i mozna zrobic ``this.toast.success(`Bucket task ${this.currentBucketTask.name} saved successfully.`)``

Reply:
Fajne :)